### PR TITLE
Feature/proxy 959

### PR DESCRIPTION
**Pull request by alex-sandrk**

### CODE REVIEW

File: public/config/index.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ const chain = {
   lmrDefaultGasLimit: process.env.LMR_DEFAULT_GAS_LIMIT,
   defaultGasPrice: process.env.DEFAULT_GAS_PRICE,
   maxGasPrice: process.env.MAX_GAS_PRICE,
+
+  localSellerProxyRouterUrl: 'http://localhost:8081',
+  localBuyerProxyRouterUrl: 'http://localhost:8082'
```
[the remark this answers is not in the document]
Comment (alex-sandrk) [reply public/config/index.js:21]: These variables are set during build of application and used to specify values that are different for environmnents(`dev`, `stg`, `prod`). 
But proxy router we gonna run on the same port for every environment. I do not see any benefits of expoing it to env variable.

File: public/main/client/proxyRouter.js [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
 
-const runProxyRouter = (config) => {
+const PROXY_ROUTER_MODE = {
+  Buyer: "buyer",
+  Seller: "seller",
+};
+
+const MODE_TO_PORTS = {
+  [PROXY_ROUTER_MODE.Buyer]: [
+    "--proxy-address=0.0.0.0:3334",
```
[the remark this answers is not in the document]
Comment (alex-sandrk) [reply public/main/client/proxyRouter.js:12]: agree that here is not the best place for hard code, will move to config/index.js but hardcoded - https://github.com/Lumerin-protocol/WalletDesktop/pull/31/files#r1052266641

[the remark this answers is not in the document]
Comment (alex-sandrk) [reply public/main/client/proxyRouter.js:12]: agree that here is not the best place for hard code, will move to config/index.js but hardcoded - https://github.com/Lumerin-protocol/WalletDesktop/pull/31/files#r1052266641

File: public/config/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (bohdan-titan) [reply public/config/index.js:21]: What about localhost? What if we will run the proxy on the same LAN? Maybe we should move to const port and concatenate proxy URL in as `{env.SellerAddress}:{constants.SELLER_PORT}` where seller port is hardcoded and address (localhost) configurable?

[the remark this answers is not in the document]
Comment (bohdan-titan) [reply public/config/index.js:21]: I guess it is a good point for future enhancements

[the remark this answers is not in the document]
Comment (alex-sandrk) [reply public/config/index.js:21]: Wallet runs Proxy router on application start. So it will be localhost for Wallet always

**Pull request merged**